freqtrade/strategy/interface.py

```diff
@@ -19,6 +19,24 @@
 logger = logging.getLogger(__name__)
 
 
+class candle_analyzed:
```

Comment:
nitpick really - Classes should be called `CandleAnalyzed` - this distinguishes it clearly a class definition when used somewhere else in the code.

Reply:
Good to know